Cython/Utility/ModuleSetupCode.c

```diff
@@ -861,11 +861,19 @@ static CYTHON_INLINE void *__Pyx_PyModule_GetState(PyObject *op)
 }
 #endif
 
+// The "static safe" variants may return NULL on static types with the Limited API on earlier versions
+// so should be used for optimization rather than where a result is required.
```

Comment:
So, maybe name it `__Pyx_PyType_TryGetSlot()` to make sure that it's not guaranteed to work in all cases/environments? "StaticSafe" isn't a very clear name for me. It suggests something like "safe to use on static types" where I usually don't know or don't want to care whether what I have in my hands might be statically defined or not. It's unclear why it wants me to care.